core/indigo-core/molecule/src/molecule_cdxml_saver.cpp

```diff
@@ -703,6 +703,235 @@ void MoleculeCdxmlSaver::saveMoleculeFragment(BaseMolecule& mol, const Vec2f& of
     _current = parent;
 }
 
+void MoleculeCdxmlSaver::addMetaData(const MetaDataStorage& meta, int id)
+{
+    const auto& meta_objects = meta.metaData();
+    for (int meta_index = 0; meta_index < meta_objects.size(); ++meta_index)
+    {
+        id++;
+        PropertiesMap attrs;
+        attrs.clear();
+        auto pobj = meta_objects[meta_index];
+        switch (pobj->_class_id)
```

Comment:
it seems no default for the switch

Reply:
Default statement on its place in the bottom of the body of the switch

Reply:
fixed